gatsby-config.js

```diff
@@ -39,6 +39,14 @@ module.exports = {
         },
       },
     },
+    {
+      resolve: `gatsby-source-ghost`,
+      options: {
+        apiUrl: `https://ghost.pennlabs.org`,
+        contentApiKey: process.env.GHOST_API_KEY,
```

Comment:
Is this in the .env.production and .env.development files? https://www.gatsbyjs.org/docs/environment-variables/#client-side-javascript

Reply:
Since the key is only used during build time, it doesn't seem like it needs to be included in the client side files.

Reply:
yep makes sense